### PR TITLE
Update PPP

### LINKED ISSUE #224: Eugene PPP

### PR DESCRIPTION
**Pull request by eugenetayyj**
## Issues 
Closes #224

### CODE REVIEW

**Review by shauneang — APPROVED** [review]

File: docs/team/eugenetayyj.md [hunk 1]
```diff
@@ -5,36 +5,50 @@ title: Eugene Tay's Project Portfolio Page
...
+    * Improved polymorphism of `Person` class to have a Has-A Relationship with `NetWorth` class.
+    * Redesigned `Tag` class's validation and constraints to only validate `POTENTIAL` and `SECURED` tags.
+<br>
 
 * **Documentation**:
     * Set Up:
-      * Set Up Continuous Integration and AY2223S1-CS2103T-T10-2 Organization.
+      * Set up AY2223S1-CS2103T-T10-2 Organization.
+      * Set Up Java Continuous Integration.
+      * Set up and link Code Coverage to team repository.
```
Comment (ReaganTan00) [inline docs/team/eugenetayyj.md:41]: I think this one is Project Management

File: docs/team/eugenetayyj.md [hunk 2]
```diff
@@ -5,36 +5,50 @@ title: Eugene Tay's Project Portfolio Page
...
+      * Introduced Legend portion.
+      * Standardise and reformat all feature write-ups.
+      * Resize and standardise all UI examples.
+      * Annotated all UI examples.
 
-* **Community**:
-    * [to be added soon]
+    * Developer Guide:
+      * Edited and improved on initial assigned client PDF feature implementation.
+      * Restructure Sequence and Activity Diagrams.
```
Comment (ReaganTan00) [inline docs/team/eugenetayyj.md:54]: Can go crazier and elaborate on things you done here

**Review by ReaganTan00 — APPROVED** [review]
LGTM, suggested some stuff

**Comment by shauneang** [comment]
lgtm

**Review by ReaganTan00 — APPROVED** [review]
LGTM

**Pull request merged**